Antidote/CallsManager.h

```diff
@@ -0,0 +1,35 @@
+//
+//  CallsManager.h
+//  Antidote
+//
+//  Created by Chuong Vu on 7/13/15.
+//  Copyright (c) 2015 dvor. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@class OCTChat;
+@class OCTCall;
+#import "AbstractCallViewController.h"
+
+@interface CallsManager : NSObject <AbstractCallViewControllerDelegate>
```

Comment:
No need to make this public. `#import "AbstractCallViewController.h"` can be moved to implementation as well.